airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -1462,7 +1529,6 @@ public String toString() {
    * your_containers_id" (ex. docker container attach 18cc929f44c8) to see the container's output
    */
   @Test
-  @Disabled
```

Comment:
Enabling this test may break some connectors and will keep lots of trash in DB. Basically, it wasn't even a test, but just a script for manual load emulation.